packages/keycodes/src/index.js

```diff
@@ -53,7 +53,7 @@ const modifiers = {
 	alt: () => [ ALT ],
 	ctrlShift: () => [ CTRL, SHIFT ],
 	shift: () => [ SHIFT ],
-	shiftAlt: () => [ SHIFT, ALT ],
+	shiftAlt: ( _isApple ) => _isApple() ? [ SHIFT, ALT ] : [ SHIFT, CTRL ],
```

Comment:
This will change some already defined shortcuts in Gutenberg, which we don't want! Those are:
⇧⌥N
Navigate to the next part of the editor (alternative).
⇧⌥P
Navigate to the previous part of the editor (alternative).

It's important they stay the same.

Reply:
Reverted this back and placed adjustments in `pressWithModifier`.